packages/video_player/tizen/src/video_player.cc

```diff
@@ -79,6 +92,35 @@ FlutterDesktopGpuSurfaceDescriptor *VideoPlayer::ObtainGpuSurface(
   return gpu_surface_.get();
 }
 
+void VideoPlayer::InitScreenSaverApi() {
+  screensaver_handle_ = dlopen("libcapi-screensaver.so", RTLD_LAZY);
+  if (!screensaver_handle_) {
+    LOG_INFO("[VideoPlayer] dlopen failed: %s", dlerror());
+    return;
+  }
+
+  screensaver_reset_timeout_ = reinterpret_cast<ScreensaverResetTimeout>(
+      dlsym(screensaver_handle_, "screensaver_reset_timeout"));
+
+  double version = atof(GetPlatformVersion().c_str());
+  if (version > 6.0) {
+    ScreensaverOverrideReset screensaver_override_reset =
+        reinterpret_cast<ScreensaverOverrideReset>(
+            dlsym(screensaver_handle_, "screensaver_override_reset"));
+
+    if (!screensaver_override_reset) {
+      LOG_INFO("[VideoPlayer] Symbol not found: %s", dlerror());
+      dlclose(screensaver_handle_);
+      return;
+    }
+    int ret = screensaver_override_reset(false);
+    if (ret != 0) {
+      throw VideoPlayerError("screensaver_override_reset failed",
+                             get_error_message(ret));
```

Comment:
Should this be treated as a critical error (fail to create a player)?

Reply:
Sorry that I don't understand what you mean....
Throw this error to let user know that screensaver_override_reset failed, it will not affect create player. Do you mean it's unnecessary to throw this error?

Reply:
Fixed.